presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixErrorCode.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.phoenix;
+
+import io.prestosql.spi.ErrorCode;
+import io.prestosql.spi.ErrorCodeSupplier;
+import io.prestosql.spi.ErrorType;
+
+import static io.prestosql.spi.ErrorType.EXTERNAL;
+
+public enum PhoenixErrorCode
+        implements ErrorCodeSupplier
+{
+    PHOENIX_ERROR(0, EXTERNAL),
+    PHOENIX_NON_TRANSIENT_ERROR(1, EXTERNAL);
+
+    private final ErrorCode errorCode;
+
+    PhoenixErrorCode(int code, ErrorType type)
+    {
+        errorCode = new ErrorCode(code + 0x0105_0000, name(), type);
```

Comment:
`0x0105` is for the Thrift connector. I registered `0x0106` for the Phoenix connector here: https://github.com/prestosql/presto/wiki/Error-Codes